### DISCUSSION TITLE
home projected and away projected

### DISCUSSION THREAD

**Discussion by mukhiil** [cwendt94/espn-api#185] [ESPN API]
Hey, first of all I just wanted to say that this package has been great!

Recently you guys added functionality in the basketball api to get the "home_projected" and "away_projected" stats from the current matchup. Even though I am in the current week, the projections are being returned as "-1" rather than an actual value. I am just curios to see if anyone else is having this problem.

Thank you!

**Comment by cwendt94** [comment]
For the `projected` scores I copied the same logic as the Football API which might not be the same for Basketball. Right now because its the start of a new matchup projected scores are not showing on ESPN right now for my league. When games start playing for this matchup I will check the data ESPN has to figure out which field the projections are stored in.
